locksmith/src/operations/lockSettingOperations.ts

```diff
@@ -52,13 +53,24 @@ export async function getSettings({
     },
   })
 
-  const res = settings || DEFAULT_LOCK_SETTINGS
+  const lockSettings = settings || DEFAULT_LOCK_SETTINGS
+
+  const eventDetails = await getEventForLock(lockAddress, network)
```

Comment:
Getting the event's settings for `replyTo` and `emailSender` and prioritizing these.